app/test/check_tx_test.go

```diff
@@ -30,8 +30,15 @@ import (
 // Here we only need to check the functionality that is added to CheckTx. We
 // assume that the rest of CheckTx is tested by the cosmos-sdk.
 func TestCheckTx(t *testing.T) {
+
+	var (
+		err  error
+		resp *abci.ResponseCheckTx
```

Comment:
there was a shadowing issue in the test also with the `err` var not being assigned with the CheckTx call below.